src/map/map.ts

```diff
@@ -256,6 +258,7 @@ export class PropertiesMap {
         this.onselect = () => {};
         this.activeChanged = () => {};
         this._selected = new Map<GUID, MarkerData>();
+        this.warnings = new Warnings;
```

Comment:
shouldn't this class take a warnings parameter to use the same one as the rest of the visualizer?

Reply:
yes was a quick hack